scvi/module/_mrdeconv.py

```diff
@@ -69,14 +78,16 @@ def __init__(
         self.n_latent = n_latent
         self.n_genes = n_genes
         self.amortization = amortization
+        self.l1_sparsity = l1_sparsity
+        self.beta_prior_weighting = beta_prior_weighting
         # unpack and copy parameters
         self.decoder = FCLayers(
             n_in=n_latent,
             n_out=n_hidden,
             n_cat_list=[n_labels],
             n_layers=n_layers,
             n_hidden=n_hidden,
-            dropout_rate=0,
+            dropout_rate=0.05,
```

Comment:
Something else that's kind of funky. Here you use 0.05, but you already changed in DestVI the other dropout to 0.05. Do you want all of them to be destvi.dropout param, or would you like only this one to be stuck to 0.05 when users change destvi.dropout?

If this is the latter option, then please add a comment explaining that the dropout of the decoder should stay small and match the CondSCVI.

Reply:
Or just remove the dropout parameters from CondSCVI and DestVI and let them fixed everywhere?

Reply:
DestVI.dropout is currently not exposed. So I think they are already fixed in destVI. I also fixed them now for CondSCVI. I have no good intuition whether they should be the same here as in CondSCVI or whether this is by chance here.

Reply:
Removed the parameter from both. Here it was working when both are the same for all three datasets. I don't have a good intuition whether it worked because all are small (0.03 was working worse though) or whether they should be the same (I wouldn't understand why they should be the same).

Reply:
I kept it in the summary string of CondSCVI for consistence with other models.